### DISCUSSION TITLE
pre-release tooltips over tab dissapeared!

### DISCUSSION THREAD

**Discussion by davintxi** [sumatrapdfreader/sumatrapdf#2801] [Q&A] [answered]
Hello, 
I did by mistake an update on the pre-release version I was using and it lost this feature that shows the file name when hovering.

I can't find an older pre-release version on my system,...the update deleted the one I was used.

There is some way to activate the tooltips over tab, or some way to get an older version of the version that still has the feature?

Thank you very much! :-)
David

**Comment by GitHubRulesOK** [comment]
Hmmm the current official 3.4.6 still has that feature

**Comment by kjk** [comment, marked as answer]
It's a bug. I rewrote tabs code a bit in pre-release and regressed tooltips. Will fix.

**Comment by kjk** [comment]
Should be fixed in pre-release 15226 or later.

**Comment by davidbstanley** [comment]
The tabs are broken for me in this latest 15226 release.  It looks like the tab name is wrapping but there is no space for that.  Also closing all tabs does not get back to the home page but leaves the last file shown.
![tool1](https://user-images.githubusercontent.com/13365534/179842877-fe0ad89a-a74b-4d5a-b0d8-6326ce191701.png)
![Tool2](https://user-images.githubusercontent.com/13365534/179843192-62312191-5df8-4539-a039-b38afb701300.png)

Once the tabs get into the state in the second image then it will not show anything else.  The only way out is to close Sumatra and start again.

I am on Win10x64.  I see this on two different machines.
